squidb/src/com/yahoo/squidb/data/ValuesStorage.java

```diff
@@ -179,6 +179,18 @@ public void put(@Nonnull String key, @Nullable Object value, boolean errorOnFail
     }
 
     @Override
+    public String toString() {
+        StringBuilder result = new StringBuilder();
```

Comment:
Minor suggestion: it might be helpful to have the specific type of the ValuesStorage in there for debugging too, along with some brackets to more clearly denote where values begin and end. How about something like:
```java
StringBuilder result = new StringBuilder(getClass().getSimpleName().toString()).append(": {\n");
// for loop here
result.append("}\n");
return result.toString();
```